### DISCUSSION TITLE
Is there support for GitHub projects (boards)

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [renovatebot/renovate#11912] [Request Help]
I am searching in the docs for a way to assign a PR created by Renovate to a GitHub project board, but I can't find it.

If it doesn't exist, are you thinking to add it knowing that GitHub is renovating the boards and they will be more popular? More info [here](https://github.com/features/issues/)

**Comment by rarkins** [comment]
There's not a way, and you're welcome to create a feature request to discuss how you think it could work

**Reply by HonkingGoose** [reply]
Do not create a new issue, we already have a feature request:

- https://github.com/renovatebot/renovate/issues/3616

**Reply by JavierSegoviaCordoba** [reply]
That issue is searching the same I am searching, having one project where I can coordinate all my repositories so I can check there all PRs for dependency updates easily, currently, I am assigning the dashboard issue to me and I have to enter in each dashboard, today I have 8 dashboards.

Hope it can be improved soon, overall when the new Projects feature is available publicly.

**Reply by rarkins** [reply]
PRs welcome

**Reply by JavierSegoviaCordoba** [reply]
Sadly, my knowledge about typescript (or web development in general) is almost zero :(

**Reply by rarkins** [reply]
Hopefully not too long until someone really needs it :)
